h2o-genmodel/src/main/java/hex/genmodel/tools/PrintMojo.java

```diff
@@ -21,10 +30,16 @@
   private static int treeToPrint = -1;
   private static int maxLevelsToPrintPerEdge = 10;
   private static boolean detail = false;
-  private static String outputFileName = null;
+  private String outputFileName = null;
```

Comment:
I cannot say I like all these being static, but we should have them all static or none